backend/internal/model/game.go

```diff
@@ -16,24 +17,34 @@ type Game interface {
 	GetScore() GameScore
 	GetScoreChannel() chan GameScore
 	SubGoal(int) error
+	UpdateRecordedShots(adapter.ShotMessage) error
+	GetRecordedShots() Shots
 }
 
 type game struct {
-	score        GameScore
-	scoreChannel chan GameScore
-	m            sync.RWMutex
+	score         GameScore
+	recordedShots Shots
+	scoreChannel  chan GameScore
+	m             sync.RWMutex
 }
 
 type GameScore struct {
 	BlueScore  int `json:"blueScore"`
 	WhiteScore int `json:"whiteScore"`
 }
 
+type Shots struct {
```

Comment:
Should it be named Shots?

Maybe ShotStats could be more verbose.

Reply:
Changed this, and method names to use ShotData instead